sdcm/sct_config.py

```diff
@@ -3153,19 +3182,21 @@ def _validate_cloud_backend_parameters(self):
             i['externalId'] for i in cloud_api_client.get_instance_types(
                 cloud_provider_id=provider_id, region_id=region_id)['instances']
         ]
-        db_instance_type = self.get('instance_type_db' if cloud_provider == 'aws' else 'gce_instance_type_db')
-        if db_instance_type not in supported_instances:
-            raise ValueError(
-                f"Database instance type '{db_instance_type}' is not supported in region '{region_name}' for "
-                f"cloud provider '{cloud_provider}'.\n"
-                f"Supported instance types: {', '.join(supported_instances)}")
-
-        rf = self.get('xcloud_replication_factor')
-        n_nodes = self.get('n_db_nodes')
-        if rf is None:
-            self['xcloud_replication_factor'] = n_nodes
-        elif rf > n_nodes:
-            raise ValueError(f"xcloud_replication_factor ({rf}) cannot be greater than n_db_nodes ({n_nodes})")
+        if not self.get("scaling"):
+            # setting db_instance_type, rf and n_nodes only for non-xcloud clusters
```

Comment:
I don't see a need to automatically set db_instance_type... SCT shouldn't be having this logic...